python/tk_framework_desktopserver/server.py

```diff
@@ -36,12 +38,37 @@ def __init__(self, port=None, debug=True, whitelist=None, keys_path=None):
         self._whitelist = whitelist or self._DEFAULT_WHITELIST
         self._debug = debug
 
-        self._logger_root = logging.getLogger("tk-framework-desktopserver")
+        self._logger_root = get_logger()
```

Comment:
Sorry, after looking at the rest of the code I understand! So ignore the previous comment :) Sorry!

Grepping the code quickly, it looks like `self._logger_root` is only used inside `Server.get_logger()` method. Could the member variable be removed completely?